### DISCUSSION TITLE
How to set default year/month in year/month selector in Date Picker?

### DISCUSSION THREAD

**Discussion by anatolyeltsov** [tusen-ai/naive-ui#3897] [Q&A]
Is it possible to do so?

For example: I want users to enter their birthday and my target audience is about 30 years old so it would be better if they could start selecting not from current year and month (2022/10) but from 1990/01 for example. And as of now the year and month in selector are always current year and month if current value is not yet set.

There is a `default-time` prop for DateTime and DateTimeRange components - maybe it would be nice to have a default-date for Date/DateRange too?

Any suggestions?

UPD: I want an input itself to be empty on init load

![Screenshot from 2022-10-20 14-07-14](https://user-images.githubusercontent.com/54027342/196933228-52fb9cbf-d408-4014-96b1-326e340b98f1.png)

**Comment by XieZongChen** [comment]
https://codesandbox.io/s/suspicious-yalow-wrqzsm?file=/src/Demo.vue
You need set `default-value`  or `v-model:value` in components init.

**Reply by anatolyeltsov** [reply]
Yeah, but this way it will be shown as input value and I need input itself to be empty

**Reply by XieZongChen** [reply]
> Yeah, but this way it will be shown as input value and I need input itself to be empty

We will consider adding this API, but the priority is not high.
